var/spack/repos/builtin/packages/elsi/package.py

```diff
@@ -64,6 +66,8 @@ def cmake_args(self):
         args += ["-DCMAKE_Fortran_COMPILER=" + self.spec["mpi"].mpifc]
         args += ["-DCMAKE_C_COMPILER=" + self.spec["mpi"].mpicc]
         args += ["-DCMAKE_CXX_COMPILER=" + self.spec["mpi"].mpicxx]
+        if "%gcc" in self.spec:
+            args += ["-DCMAKE_Fortran_FLAGS_RELEASE=-O2 -DNDEBUG"]
```

Comment:
Can you comment why you need to override these defaults? What was used previously, and why was it wrong / suboptimal?

Reply:
@alalazo 
Thanks for the review. I have added a comment to the code. This process was added because the compiler stopped abnormally when built with gcc compiler 8.5.0.

Reply:
Is the issue fixed by newer `gcc` versions?

Reply:
@kinagaki-fj